openvdb/CHANGES

```diff
@@ -15,6 +15,9 @@ Version 6.0.1 - In development
     - Fixed a bug where the Points Convert SOP was incorrectly ignoring
       point attributes with the same name as an existing point group.
 
+    Python:
+    - Added basic PointDataGrid support to the Python module.
```

Comment:
Maybe it would be good to clarify that, for now, support is limited (more or less) to I/O and metadata operations.

Reply:
@pcucka - sure, done.

Reply:
Thanks!